### PR TITLE
Add Content Feature

### PR DESCRIPTION
**Pull request by Arique1104**

### CODE REVIEW

File: filename.rb [hunk 1]
```diff
@@ -1 +1,3 @@
+"Hello Wilmer!"
```
Comment (priyapower) [inline filename.rb:1]: I am commenting. Nice nice nice

Comment (Arique1104) [reply filename.rb:1]: Thanks for looking at my code!

File: filename.rb [hunk 2]
```diff
@@ -1 +1,3 @@
...
+
```
Comment (priyapower) [inline filename.rb:2]: Why is there white space?

Comment (Arique1104) [reply filename.rb:2]: For readability.

**Pull request merged**